server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.net.InetAddress;
+
+public enum HotRodClientTestingUtil {
```

Comment:
Right, but you end up with a lot of test util classes doing a similar thing. I understand that you can't depend on HRServer but the `HotRodClientTestingUtil` could be modified to avoid doing that too and that way reuse it in both places. Not urgent but something to keep in mind.